src/content/en/fundamentals/security/credential-management/save-forms.md

```diff
@@ -0,0 +1,145 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2016-11-08 #}
+{# wf_published_on: 2016-11-08 #}
+
+# Save Credentials from Forms {: .page-title }
+
+{% include "web/_shared/contributors/agektmr.html" %}
+{% include "web/_shared/contributors/megginkearney.html" %}
+
+Forms that let users sign-up, sign-in, or change passwords are great chances
+for you to store user’s credential information. 
+
+<div class="attempt-right">
+  <figure>
+    <video src="animations/credential-management-smaller.mov" style="max-height: 400px;" autoplay muted loop controls></video>
+    <figcaption>Save Credentials from sign-in forms</figcaption>
+  </figure>
+</div>
+
+To store user credentials from forms:
+
+1. Check form includes `autocomplete` attributes.
+2. Interrupt a form submission.
+3. Authenticate through AJAX.
+4. Store the credential.
+5. Update the UI or proceed to the personalized page.
+
+## Include autocomplete in form
+
+Before moving forward,
+check if your form includes `autocomplete` attributes.
+This helps the Credential Management API find the `id` and `password`
+from the form and construct a credential object.
+
+This also helps browsers not supporting the Credential Management API
+to understand its semantics.
+Adding autocomplete is a MUST.
+Learn more about the topic from
+[an article](https://cloudfour.com/thinks/autofill-what-web-devs-should-know-but-dont/) by
+[Jason Grigsby](https://medium.com/@grigs).
+
+    <form id="signup" method="post">
+     <input name="email" type="text" autocomplete="username email">
+     <input name="display-name" type="text" autocomplete="name"> 
+     <input name="password" type="password" autocomplete="new-password">
+     <input type="submit" value="Sign Up!">
+    </form>
+
+## Interrupt form submission
+
+Interrupt the form submission event when user presses the submit button,
+and prevent the default behavior.
+
+    var f = document.querySelector('#signup');
+    f.addEventListener('submit', e => {
+     e.preventDefault();
+
+By preventing a page transition,
+you can retain the credential information while verifying its authenticity.
+
+## Send a request via AJAX
+
+Send a request via AJAX and respond with profile information,
+for example, in JSON format.
+
+The following is pseudo AJAX
+(replace with a [`fetch()`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) call):
```

Comment:
We definitely need explanation about how to use `fetch()`. I would revert the section you ditched "Authenticate with a username and password" somewhere.